@utahdts/utah-design-system/react/components/containers/tabs/TabGroup.jsx

```diff
@@ -38,12 +50,39 @@ export function TabGroup({
           setTabGroupState((draftState) => { draftState.selectedTabId = tabId; });
         }
       },
+      navigateNext: () => {
+        const index = findCurrentTabIndex();
+        if (index !== (tabGroupState.tabs.length - 1)) {
+          navigateTab(tabGroupState?.tabs?.[index + 1] || null);
+        } else {
+          navigateTab(tabGroupState?.tabs?.[0] || null);
+        }
+      },
+      navigatePrevious: () => {
+        const index = findCurrentTabIndex();
+        if (index !== 0) {
```

Comment:
modulo math may make this simpler?
`const nextIndex = (index + tabGroupState.tabs.length - 1) % tabGroupState.tabs.length;`